tsfc/fem.py

```diff
@@ -319,9 +317,10 @@ def callback(key):
         table = ctx.tabulation_manager[key]
         if len(table.shape) == 1:
             # Cellwise constant
-            row = gem.Literal(table)
+            row = table
         else:
-            table = ctx.index_selector(lambda i: gem.Literal(table[i]), mt.restriction)
+            table = ctx.index_selector(lambda i: gem.partial_indexed(table, (i,)),
```

Comment:
Oh, but previously `table` was a numpy array and now it's a `Literal`.  So this would have to be `ctx.index_selector(lambda i: gem.Literal(table.array[i]), mt.restriction)` I think.